shortcodes/ucf-faq-list-shortcode.php

```diff
@@ -68,15 +70,30 @@ public static function shortcode( $atts ) {
 			$posts = get_posts( $args );
 			$items = array();
 
-			foreach( $posts as $post ) {
+			foreach ( $posts as $post ) {
 				$topics = wp_get_post_terms( $post->ID, 'topic' );
 
-				foreach( $topics as $topic ) {
+				foreach ( $topics as $topic ) {
 					$items[$topic->name][] = $post;
 				}
 			}
 
-			return UCF_FAQ_List_Common::display_faqs( $items, $atts['layout'], $atts );
+			$faqs = UCF_FAQ_List_Common::display_faqs( $items, $atts['layout'], $atts );
+
+			$related_faqs_html = "";
+
+			if ( $atts['tags'] !== '' ) {
+
+				$related_posts = UCF_FAQ_Common::get_related_faqs( $atts['tags'], $items );
+
+				$related_faqs_html = '<h2 class="h4 mt-4">Related FAQs</h2>';
```

Comment:
This heading + classes should probably be configurable like the question and topic headings